clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/memory/InMemoryStorageEngine.java

```diff
@@ -15,8 +15,12 @@
  */
 public class InMemoryStorageEngine extends AbstractStorageEngine<InMemoryStoragePartition> {
   public InMemoryStorageEngine(VeniceStoreVersionConfig storeDef) {
+    this(storeDef.getStoreVersionName());
+  }
+
+  public InMemoryStorageEngine(String versionName) {
```

Comment:
Seems we will re-use `InMemoryStorageEngine` all the time and drop the partitions when we do version swap, as version is changed, but the previous `versionName` is still kept inside this class, is it better to just hold storeName inside this class?

Reply:
I'm not totally sure I follow.  The 'versionName' argument is just passed to the super, where it's held.  Interestingly, the super has the parameter labeled as 'storeName'.  So the versionName here is 'kind of' a misnomer perhaps?  Admittedly, we use these storage engines on a per version basis.  So it's kind of a toss up to say which is right imo.